### PR TITLE
Replace unexpected-react with unexpected-htmllike

### PR DESCRIPTION
**Pull request by sunesimonsen**

### CODE REVIEW

File: test/__snapshots__/to-contain.spec.js.snap [hunk 1]
```diff
@@ -1,14 +1,63 @@
...
+      </li>
+    </ul>
+  </div>
+</Fixture>
+to contain <User id={4} />
 
 the best match was
 <Fixture // should be <User
    // missing id={4}
 />"
```
Comment (sunesimonsen) [inline test/__snapshots__/to-contain.spec.js.snap:29]: This seems to be a bug in htmllike, if I call children on the wrapper then a get a really nice diff.

**Comment by sunesimonsen** [comment]
Startup time is still awful, but the tests are fast:

```
 PASS  test/to-contain.spec.js
  to-contain
    <ReactElement>
      ✓ passes when the actual matches the expected (28ms)
      ✓ fails when the actual does not match the expected (72ms)
      when negated
        ✓ passes negated when the actual does not match the expected (2ms)
        ✓ fails when the actual does match the expected (26ms)
    <selector>
      ✓ passes when the actual matches the expected (7ms)
      ✓ fails when the actual does not match the expected (17ms)

 PASS  test/to-have-state-satisfying.spec.js
  to-have-state-satisfying
    <assertion>
      ✓ passes when the actual matches the expected (14ms)
      ✓ fails when the actual does not match the expected (67ms)
      when negated
        ✓ passes when the actual does not match the expected (2ms)
        ✓ fails when the actual does match the expected (14ms)
    <object>
      ✓ passes when the passed matches the expected (6ms)
      ✓ fails when the actual does not match the expected (36ms)
      with state as null
        ✓ passes when the actual matches the expected (1ms)

 PASS  test/to-satisfy.spec.js
  to-satisfy
    when it is a subtree
      ✓ passes when the actual matches the expected (5ms)
      ✓ fails when the actual does not the expected (53ms)
      ✓ fails when the actual not matching the expected type (10ms)

 PASS  test/to-have-class.spec.js
  to-have-class
    ✓ passes when the actual matches the expected (5ms)
    ✓ passes negated when the actual does not match the expected (2ms)
    ✓ fails when the actual does not match the expected (46ms)
    when rendering a Component
      ✓ passes when the actual matches the expected (3ms)
    when deeply nested
      ✓ passes when the actual matches the expected (2ms)

 PASS  test/queried-for.spec.js
  queried-for
    ✓ passes when the actual matches the expected (5ms)
    ✓ fails when the actual does not match the expected (49ms)

 PASS  test/to-render-text.spec.js
  to-render-text
    ✓ passes when actual matches the expected (3ms)
    ✓ passes negated when actual does not match the expected (2ms)
    when nested
      ✓ passes when actual matches the expected (5ms)
    <assertion>
      ✓ passes when actual matches the expected (3ms)
      ✓ fails when actual does not match the expected (54ms)

 PASS  test/to-have-props-satisfying.spec.js
  to-have-props-satisfying
    ✓ passes when the passed matches the expected (4ms)
    ✓ fails when the actual does not match the expected (54ms)

 PASS  test/to-have-type.spec.js
  to-have-type
    string
      ✓ passes when actual matches expected (2ms)
      ✓ passes negated when actual does not match the expected (2ms)
      ✓ fails when actual does not match expected (34ms)
      ✓ fails when negated actual matches expected (9ms)
    function
      ✓ passes when actual matches expected (2ms)
      ✓ passes negated when actual does not match the expected (1ms)
      ✓ fails when actual does not match expected (13ms)
      ✓ fails when negated actual matches expected (16ms)

 PASS  test/to-be-checked.spec.js
  to-be-checked
    ✓ passes when the actual matches the expected (8ms)
    ✓ passes negated when the actual does not match the expected (2ms)
    ✓ fails when the actual does not match the expected (35ms)

 PASS  test/to-exist.spec.js
  to-have-props-satisfying
    ✓ passes when the actual matches the expected (3ms)
    ✓ passes negated when the actual matches the expected (2ms)
    ✓ fails when the actual does not match the expected (32ms)
    ✓ fails when the actual negated matches the expected (7ms)
    with queried for
      ✓ fails when the actual does not match the expected (7ms)

 PASS  test/to-have-props.spec.js
  to-have-props
    ✓ passes when the actual matches the expected (3ms)
    ✓ fails when the actual does not match the expected (35ms)
    when negated
      ✓ passes when the actual does not match the expected (3ms)
      ✓ fails when the actual does match the expected (9ms)

 PASS  test/to-have-context.spec.js
  to-have-context-satisfying
    ✓ passes when the passed matches the expected (6ms)
    ✓ fails when the actual does not match the expected (44ms)

 PASS  test/to-have-context-satisfying.spec.js
  to-have-context-satisfying
    ✓ passes when the passed matches the expected (4ms)
    ✓ fails when the actual does not match the expected (53ms)

 PASS  test/when-setting-state.spec.js
  when-setting-state
    ✓ passes when actual matches the expected (4ms)
    when updating props
      ✓ updates the className (5ms)
      ✓ previous className is no longer rendered (2ms)

 PASS  test/when-setting-props.spec.js
  when-setting-props
    ✓ passes when actual matches the expected (5ms)
    when updating props
      ✓ updates the className (3ms)
      ✓ previous className is no longer rendered (2ms)

 PASS  test/when-setting-context.spec.js
  when-setting-context
    ✓ passes when actual matches the expected (2ms)
    when updating props
      ✓ updates the className (3ms)
```

File: package.json [hunk 2]
```diff
@@ -10,9 +10,10 @@
     "test:watch": "jest --watch",
     "lint": "eslint .",
     "format": "prettier --write {config,src,test}/**/*.js",
-    "ci": "yarn format && yarn lint && yarn test",
+    "ci": "yarn format && yarn lint && yarn test -i",
```
Comment (albertfdp) [inline package.json:13]: Why should we run them in band in CI?

**Review by albertfdp — APPROVED** [review]
Looks great!

File: package.json (hunk 2 above)
Comment (sunesimonsen) [reply package.json:13]: Travis runs much slower with parallel Jest 😢

**Pull request merged**